content/en/docs/concepts/configuration/secret.md

```diff
@@ -359,6 +359,38 @@ echo 'MWYyZDFlMmU2N2Rm' | base64 --decode
 1f2d1e2e67df
 ```
 
+#### Editing a Secret
```

Comment:
I suggest: add a new, small task page about this, and link to the task page from this point in the concept page about Secrets. Also link to that task page in the Further reading.

Reply:
For this specific PR, just adding this content looks good to me. We may need a followup PR to split a task page from this huge page.